src/sentry/static/sentry/app/utils/discover/fieldRenderers.tsx

```diff
@@ -390,71 +388,48 @@ type SpecialFunctions = {
  * or they require custom UI formatting that can't be handled by the datatype formatters.
  */
 const SPECIAL_FUNCTIONS: SpecialFunctions = {
-  user_misery_prototype: data => {
-    const uniqueUsers = data.count_unique_user;
-    let miseryField: string = '';
-    let userMiseryField: string = '';
-    for (const field in data) {
-      if (field.startsWith('user_misery_prototype')) {
-        miseryField = field;
-      } else if (field.startsWith('user_misery')) {
-        userMiseryField = field;
-      }
-    }
-
-    if (!miseryField) {
-      return <NumberContainer>{emptyValue}</NumberContainer>;
-    }
-
-    const miserableUsers = userMiseryField ? data[userMiseryField] : undefined;
-    const userMisery = miseryField ? data[miseryField] : undefined;
-
-    const miseryLimit = parseInt(miseryField.split('_').pop() || '', 10);
-
-    return (
-      <BarContainer>
-        <UserMiseryPrototype
-          bars={10}
-          barHeight={20}
-          miseryLimit={miseryLimit}
-          totalUsers={uniqueUsers}
-          userMisery={userMisery}
-          miserableUsers={miserableUsers}
-        />
-      </BarContainer>
-    );
-  },
   user_misery: data => {
-    const uniqueUsers = data.count_unique_user;
     let userMiseryField: string = '';
+    let countMiserableUserField: string = '';
     for (const field in data) {
       if (field.startsWith('user_misery')) {
         userMiseryField = field;
+      } else if (field.startsWith('count_miserable_user')) {
+        countMiserableUserField = field;
       }
     }
+
     if (!userMiseryField) {
       return <NumberContainer>{emptyValue}</NumberContainer>;
     }
 
+    const uniqueUsers = data.count_unique_user;
     const userMisery = data[userMiseryField];
-    if (!uniqueUsers && uniqueUsers !== 0) {
-      return (
-        <NumberContainer>
-          {typeof userMisery === 'number' ? formatFloat(userMisery, 4) : emptyValue}
-        </NumberContainer>
-      );
-    }
 
     const miseryLimit = parseInt(userMiseryField.split('_').pop() || '', 10);
 
+    let miserableUsers: number | undefined;
+
+    if (countMiserableUserField) {
+      const countMiserableMiseryLimit = parseInt(
+        countMiserableUserField.split('_').pop() || '',
+        10
+      );
```

Comment:
`countMiserableMiseryLimit` and `miseryLimit` has the potential to be `NaN` when we see bad data. Can we guard against this just in case?